assets/css/amp-editor-story-blocks.css

```diff
@@ -24,6 +25,18 @@
 
 -------------------------------------------------------------------*/
 
+/*
+ * 0. General editor layout styles.
+ */
+@media screen and (min-width: 1280px) {
+	.edit-post-visual-editor {
+		max-width: 1024px;
+		margin: 0 auto;
+	}
+	#amp-story-editor {
+		min-width: 800px;
+	}
```

Comment:
@swissspidy On this: https://github.com/ampproject/amp-wp/pull/2001#pullrequestreview-217233133
I'm not actually seeing a difference between how it is right now and how it was with editor styles. Note that I'm using twentynineteen theme and am not using a wide screen.

Would you mind checking if this displays normally in your screen and adjust if not? Setting these widths also moves the Block Navigation a little bit and I know there was some discussion on where it should be displayed, I might have moved it incorrectly.